java-logging-appinsights/build.gradle

```diff
@@ -27,6 +27,7 @@ bintray {
   key = System.getenv('BINTRAY_KEY')
   publications = ['Insights', 'Azure']
   publish = true
+  override = true
```

Comment:
sounds risky

Reply:
🤔, what if i remove `Azure` from publications? it is already published. version has not changed. only needed because microsoft does not provide agent from maven repos

Reply:
I would be happy with that